linux/devdoc/threadpool_linux_requirements.md

```diff
@@ -145,30 +154,43 @@ static void threadpool_dispose(THREADPOOL* threadpool)
 
 **SRS_THREADPOOL_LINUX_07_015: [** `threadpool_destroy` shall destroy the SRW lock by calling `srw_lock_destroy`. **]**
 
+[Will be moved from threadpool_close.]
+**S_R_S_THREADPOOL_LINUX_07_089: [** `threadpool_close` shall signal all threads `threadpool` is closing by calling `InterlockedHL_SetAndWakeAll`.  **]**
+
+[Will be moved from threadpool_close.]
+**S_R_S_THREADPOOL_LINUX_07_027: [** `threadpool_close` shall join all threads in the `threadpool`. **]**
+
 **SRS_THREADPOOL_LINUX_07_016: [** `threadpool_destroy` shall free the memory allocated in `threadpool_create`. **]**
 
 ### threadpool_open
 
 ```C
 MOCKABLE_FUNCTION(, int, threadpool_open, THANDLE(THREADPOOL), threadpool);
 ```
-
+Note: threadpool_open will be deprecated and threadpool_create will perform additional tasks of threadpool_open. This function will exist until all the libraries calling this API are modified to use only threadpool_create.
```

Comment:
>threadpool_create [](http://example.com/codeflow?start=204&length=17)

backticks. Here and wherever else needed.

Reply:
Okay